examples/registrar_template.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This is very useful template to register the workload registrar entry. Thank you

Perhaps, we can also add a multi-entry example? E.g.
```
{
  "entries": [
    {
    "spiffe_id": {
      "trust_domain": "example.org",
      "path": "/sample1"
    },
    "selectors": [
        {
          "type": "k8s",
          "value": "ns:default"
        },
        {
          "type": "k8s",
          "value": "container-image:ubuntu"
        }
      ],
      "admin": false 
    },

    {
    "spiffe_id": {
      "path": "/sample2"
    },
    "selectors": [
        {
          "type": "k8s",
          "value": "sa:operator"
        },
        {
          "type": "k8s",
          "value": "container-name:my-operator"
        }
      ],
      "admin": true,
      "ttl": 600
    }
  ]
}
```